src/legacy/core_plugins/kibana/public/dashboard/panel/panel_error.tsx

```diff
@@ -17,17 +17,28 @@
  * under the License.
  */
 
+import { EuiIcon, EuiSpacer, EuiText } from '@elastic/eui';
+// import MarkdownIt from 'markdown-it';
 import React from 'react';
 
+// TODO: User Markdown
```

Comment:
I would recommend not to fix that in this PR. Looking at the diff, that has been broken beforehand, and we should just check how we want to fix that (where do the errors actually pop up? can we maybe use JSX directly instead of markdown?). Could you please open a separate issue for that?

Reply:
Sure, I'll make an issue